contracts/README.md

```diff
@@ -9,14 +9,14 @@ On chain components of the [Umbra protocol](../README.md).
 Contract development uses the OpenZeppelin tools and test framework. All are installed and exercised via `npm`. Development has been tested with:
 
 * node.js v12.16.1 or later
-* npm 6.13.4 or later
+* yarn 1.22.0 or later
 
 ### Usage
 
-From the current working directory, run `npm install`. Afterwards run:
+From the current working directory, run `yarn install`. Afterwards run:
 
-* `npm run compile` — To compile all contracts
-* `npm run test` — To run all tests
+* `yarn compile` — To compile all contracts
```

Comment:
I'd recommend updating the `compile` script to specify a solc version and the evm-target, just because it's good to be explicit and it's not clear how the above settings were selected

Reply:
we've got this in the OZ config file:

https://github.com/ScopeLift/umbra-protocol/pull/59/files#diff-eeaaa081c313f087ef845b5544746432R22